src/components/GettingStartedModal/ChooseSetStep.tsx

```diff
@@ -0,0 +1,111 @@
+import { ChangeEvent, FormEvent, ReactElement, useEffect , useState} from "react";
+import { NavigationButtons, Step, StepProps } from ".";
+import { Button } from "../Button";
+import {
+  isPhoneticsPreference,
+  PhoneticsPreference,
+  PREFERENCE_LITERATES,
+} from "../../state/reducers/phoneticsPreference";
+import { GROUPS, isGroupId } from "../../state/reducers/groupId";
+import { SectionHeading } from "../SectionHeading";
+import { CollectionDetails } from "../CollectionDetails";
+
+import { collections, VocabSet } from "../../data/vocabSets"; 
+import { useUserStateContext } from "../../providers/UserStateProvider";
+import { TermsByProficiencyLevelChart } from "../TermsByProficiencyLevelChart";
+
+export const ChooseSetStep: Step = {  
+  title: "Choose your first set",
+  commitState: ({ collectionId }, {setUpstreamCollection}) => {
+    if (collectionId !== undefined){
+      setUpstreamCollection(collectionId);
+    }
+  },
+  Component: PhoneticsStepComponent,
+};
+
+function PhoneticsStepComponent({
```

Comment:
This should be given an appropriate name, eg. ChooseSetStep or something.

Reply:
Maybe I don't understand this one, but doesn't this make more sense than chooseSetStep? This step lets the user decide which phonetics to use.

Reply:
I think this code was copy pasted but look at the filename!

Reply:
Oh whoops! I've changed this to ChooseSetStepComonent.